script/check-size

```diff
@@ -149,6 +161,39 @@ size_nrf52840()
         && ./bootstrap \
         && make -f examples/Makefile-nrf52840 "${flags[@]}")
 
+    # rename the generated files to be ready for size-report
+    # shellcheck disable=SC2011
```

Comment:
**clarification for the shellcheck disable:**
shellcheck suggested to replace `ls` with `find .. -print0 | ...` so as to handle file names with special characters correctly. 
But I think it's fine here. BTW, it seems quite difficult to write the command in the `find .. -print0 | ...` way.